src/main/java/org/opentripplanner/routing/impl/ATLFareServiceImpl.java

```diff
@@ -0,0 +1,391 @@
+package org.opentripplanner.routing.impl;
+
+import com.google.common.collect.Lists;
+import org.opentripplanner.model.Route;
+import org.opentripplanner.routing.core.Fare;
+import org.opentripplanner.routing.core.FareRuleSet;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+
+public class ATLFareServiceImpl extends DefaultFareServiceImpl {
+    public static final float DEFAULT_TEST_RIDE_PRICE = 3.49f;
+    public boolean IS_TEST;
+
+    public static final String COBB_AGENCY_ID = "2";
```

Comment:
Perhaps I don't want to hear the answer but how stable are these IDs?

Reply:
They are likely not super stable but I think stabilization of these IDs is a separate conversation that I am not a part of but have heard bits of in the tech calls. @demory